srcpkgs/kodi-addon-pvr-zattoo/template

```diff
@@ -14,7 +14,7 @@ maintainer="teldra <teldra@rotce.de>"
 license="GPL-2.0-only"
 homepage="https://github.com/rbuehlma/pvr.zattoo"
 distfiles="https://github.com/rbuehlma/pvr.zattoo/archive/${version}-${_kodi_release}.tar.gz"
-checksum=25dc4ea72b5315a65286ba606e6e26d281fb8eb830d5e0164b66b8d7ae90319a
+checksum=07d96b1b0c3b464c0bf02279fa8cab9ccdf675344129f3a970b14bafce225fb4
 make_check=no # No target to "make test"
```

Comment:
This should be a harmless warning, can we remove this line?

Reply:
> This should be a harmless warning, can we remove this line?

Yes.